billing_agreements.go

```diff
@@ -40,3 +40,31 @@ func (c *Client) CreateBillingAgreementToken(
 
 	return billingAgreementToken, nil
 }
+
+// CreateBillingAgreementFromToken - Use this call to create a billing agreement
+// Endpoint: POST /v1/billing-agreements/agreements
+func (c *Client) CreateBillingAgreementFromToken(
```

Comment:
Seems like this endpoint is deprecated.

https://developer.paypal.com/docs/api/payments.billing-agreements/v1/

can we use the subscriptions endpoint? or does it make sense to add this.

![Uploading image.png…]()

Reply:
That's for /v1/payments/billing-agreements, rather than /v1/billing-agreements/agreements. [Here](https://developer.paypal.com/docs/api/payments.billing-agreements/v1/#billing-agreements_post) is the document for the endpoint I'm using.

Reply:
ahh makes sense. Still seems like it is v1/payments/billing-agreements without the /agreement?

And I don't see the docs show that you can accept token? or am I looking at the wrong thing

Reply:
It seems like how it's set up for the other endpoints, there's one file per path of a type like v1/payments/ and one function for each path of a type like v1/payments/agreements.

Also, the endpoint returns something that looks like this where the "B-...." value is the billing agreement. I've simulated this in Postman using our API credentials and found it successful.
<img width="666" alt="Screen Shot 2021-07-12 at 3 49 44 PM" src="https://user-images.githubusercontent.com/48696097/125360223-ca782300-e328-11eb-8e99-f85ce0d1a06f.png">

Reply:
@cameronjarnot1 I guess my overall question is where is the documentation for `v1/payments/billing-agreements/agreements`? The paypal docs oage here suggests posting to `/v1/payments/billing-agreements`

![image](https://user-images.githubusercontent.com/678239/125395181-b1d72f80-e35f-11eb-9eb0-848ffc2574c4.png)

If I am looking for the docs in the wrong place could you please update the method with a link in the comments to the doc.

Reply:
Ok sorry my bad, i was confused about the documentation. You're right that that's the correct documentation, but when we talked to PayPal, they recommended using that endpoint for the channel initiated billing despite the fact that it says it's deprecated.